Cargo.toml

```diff
@@ -32,8 +32,11 @@ alloy-provider = "0.6.1"
 erc6492 = { git = "https://github.com/reown-com/erc6492.git", branch = "main" }
 
 [profile.release]
-opt-level = 'z'
+debug = true
 lto = true
+opt-level = 0
```

Comment:
@chris13524 do you have an opinion on that?

Reply:
LGTM